ci/jobs/ci-redmine-bugzilla.yaml

```diff
@@ -0,0 +1,62 @@
+# This Jenkins job runs a script which performs automation related to Redmine
+# and Bugzilla. See https://github.com/pulp/pulp_packaging/ci/redmine_bugzilla.py
+
+- job-template:
+    name: 'redmine-bugzilla-automation'
+    defaults: ci-workflow-runtest
+    node: 'f20-np'
+    scm:
+        - git:
+            url: 'https://github.com/bmbouter/pulp_packaging.git'
+            branches:
+                - origin/141-add-bug-automation
+            basedir: pulp_packaging
+            skip-tag: true
+            wipe-workspace: false
+    triggers:
+        - timed: "0,30 * * * *"
+    wrappers:
+        - credentials-binding:
+            - file:
+                credential-id: 099bc04b-80b6-4cd6-ad1d-11908641f539
+                variable: REDMINE_BUGZILLA_CONF
+        - timeout:
+            # Timeout in minutes
+            timeout: 30
+            timeout-var: 'BUILD_TIMEOUT'
+            fail: true
+    builders:
+        - shell: |
+            #!/bin/bash
+            git config --global user.email "pulp-infra@redhat.com"
+            git config --global user.name "pulpbot"
+            set -x
+            env
+
+            cd $WORKSPACE/
+
+            sudo pip install --upgrade git+https://github.com/bmbouter/python-bugzilla.git@expand-external-tracker-support
```

Comment:
It really should be `sudo pip install python-bugzilla` but we can't do that until my changes are accepted into a release that gets put on PyPi. I'm contributing them back to python-bugzilla, but until then it needs to stay like it is. I also forked from my private bmbouter account because it's not really a Pulp repo.